### DISCUSSION TITLE
Βιογραφικό Β

### DISCUSSION THREAD

**Discussion by e-liouta** [upatras-hci/iv#81] [Q&A]
Για το Βιογραφικό Β μέχρι στιγμής έχω κάνει τα παρακάτω:

• Έφτιαξα ένα νέο cv που μπορείτε να δείτε [εδώ](https://e-liouta.github.io/cv/)
Το συγκεκριμένο cv έχει ένα αρχείο .yml το οποίο και περιέχει τις βασικές πληροφορίες μου. Το αρχείο αυτό βρίσκεται [εδώ](https://github.com/e-liouta/cv/blob/main/_data/data.yml)

Στη συνέχεια έφτιαξα ένα github action το οποίο μετατρέπει το yaml αυτό αρχείο μου σε pdf. Καθώς έδειξε η πορεία για κάποιο λόγο αυτό δεν υποστηρίζεται οπότε και στη συνέχεια αυτό που έκανα είναι να μετατρέπω το yml σε md και στη συνέχεια το md σε pdf. Αυτό και πάλι σε δούλεψε.

Στη συνέχεια δοκίμασα να κάνω τις ίδιες εντολές σε ένα shell script όμως και αυτό είχε παρόμοια αποτελέσματα.

Τέλος δοκίμασα να παράγω το pdf από το html αρχείο.

Να πω σε αυτό το σημείο ότι όταν λέω ότι δε δουλεύει εννοώ σε επίπεδο CI γιατί τοπικά όλες οι εντολές δούλευαν όμως οι περισσότερες εντολές μου παρήγαγαν pdf τον κώδικα σε plain text μέσα στο pdf. 

Έτσι στη συνέχεια αποφάσισα να δημιουργήσω ένα .tex αρχείο όμως μέχρι στιγμής ούτε αυτό έχει δουλέψει.

Φυσικά μπορείτε να δείτε τι γίνεται στο [repo μου](https://github.com/e-liouta/cv)
και το [ιστορικό με τα commits](https://github.com/e-liouta/cv/commits/main) για να καταλάβετε καλύτερα τη λογική που ακολούθησα.

**Comment by epidrome** [comment]
πολύ καλή προσπάθεια, χαίρομαι που διάλεξες το μορφότυπο από έναν παλιό φοιτητή στο ιόνιο

από το πιο πρόσφατο σφάλμα στην κονσόλα φαίνεται πως δεν έχεις δώσει εντολή να εγκατασταθεί το Jekyll οπότε κολλάει εκεί και δεν πάει στο επόμενο βήμα, ενδέχεται βέβαια να έχει πολλά ακόμη λάθη
https://github.com/e-liouta/cv/actions/runs/4156774066/jobs/7190792840
